sql/src/main/java/org/apache/druid/sql/calcite/planner/PlannerFactory.java

```diff
@@ -163,6 +164,7 @@ private FrameworkConfig buildFrameworkConfig(PlannerContext plannerContext)
         .typeSystem(DruidTypeSystem.INSTANCE)
         .defaultSchema(rootSchema.getSubSchema(druidSchemaName))
         .sqlToRelConverterConfig(sqlToRelConverterConfig)
+        .costFactory(new DruidVolcanoCost.Factory())
```

Comment:
You have the if statement below to add this if it's in DECOUPLED mode, but it would appear as though you are adding it in all instances as well.  It's interesting that all tests continue to pass, which should give us confidence to just attach this in all cases, but let's be conservative and minimize impact to other modes of planning for now, so I think we can remove this.

Reply:
Good catch, yeah the new cost model doesn't seem to affect the tests. But agree to keep this gated for now, fixed it